modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesRecoveryTask.java

```diff
@@ -100,6 +99,7 @@ public void run() {
     public void executeNow() {
```

Comment:
I think it was added for some kind of an encapsulation. Let's not change it. @Asitha what do you think?